### DISCUSSION TITLE
Welcome to disciple-tools-quick-comments Discussions!

### DISCUSSION THREAD

**Discussion by prykon** [prykon/disciple-tools-quick-comments#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oldvill** [comment]
Hi. Love this addition. Thank you! 
Request/Idea: currently you only have one button that says, "Create and post quick comment". What happens if I want to Create several comments without posting?
So, can you add a button that simply says, "Create". I would like to go in and add several "quick comments" without having to post each one. Make sense?

**Reply by prykon** [reply]
Hi @oldvill !

Glad you find the addition useful :)

The feature was created under the premise that "too many quick comments aren't that quick to sort through". This means that if you have, for instance, over 20 quick comments, it would be quicker to write the comment manually than to find it in the dropdown. This is why a bulk quick comment generation feature wasn't added to the plugin. Users are, however, able to add all the quick comments they want.

Thanks for your feedback!

**Comment by oldvill** [comment]
Thank you for the feedback.
I’m not sure I agree but I definitely get the reasoning.
Regardless, thanks for the feature!
